tensorflow/tools/ci_build/mac/gpu/pip/run.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ==============================================================================
+#
+# Usage:
```

Comment:
OK. I still have some remaining concerns. So it seems that this file is meant to replace pip.sh on mac. But this script doesn't do certain things that are done by pip.sh. For example, it doesn't do renaming of the pip files as here:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/ci_build/builds/pip.sh#L167

If you add that to this file, it'll further increase the amount of duplication. So maybe instead of creating a new file, it will be better to refactor pip.sh.